Druid-Tests/DruidTestRTLCompiler.class.st

```diff
@@ -3260,24 +3249,21 @@ DruidTestRTLCompiler >> gen_pushLiteralConstantBytecode [
 	"AutoGenerated by Druid"
 
 	| currentBlock |
-	self MoveCq: ((methodObj + 8) + ((1 + LiteralStart) << 3)) R: TempReg.
-	self MoveM64: 0 r: TempReg R: TempReg.
-	self ssPushRegister: TempReg.
+	self genMoveConstant: methodObj R: TempReg.
+	self ssPushBase: TempReg offset: 1 + LiteralStart << 3 + 8.
 	^ 0
```

Comment:
Is it the expected behaviour?

Reply:
Yes, this is many super very hyper recontra great